deltachat-ios/Controller/GroupChatDetailViewController.swift

```diff
@@ -401,21 +401,20 @@ extension GroupChatDetailViewController: UITableViewDelegate, UITableViewDataSou
             !isMemberManagementRow(row: row) &&
             getGroupMemberIdFor(row) != currentUser.id {
             // action set for members except for current user
-            let delete = UITableViewRowAction(style: .destructive, title: String.localized("remove_desktop")) { [unowned self] _, indexPath in
-
-                let contact = self.getGroupMember(at: row)
+            let delete = UITableViewRowAction(style: .destructive, title: String.localized("remove_desktop")) { [weak self] _, indexPath in
+                guard let strongSelf = self else { return }
```

Comment:
In swift 4.2 the syntax `guard let self = self else { return } ` was introduced to check if self is nil or not

Reply:
personally I like that more than a `strongSelf` variable

Reply:
copied the strongSelf pattern from signal, did not know `guard let self = self else { return }` is possible - but in, fact, we were already using it at two places :)

yip, i will do a commit that changes that.

ftr, this is the discussion about it https://github.com/apple/swift-evolution/blob/master/proposals/0079-upgrade-self-from-weak-to-strong.md